game/card.go

```diff
@@ -11,19 +12,16 @@ type Card struct {
 	AddsTemporaryEffect  bool
 	Bloodthirst          int
 	CastingCost          *CastingCost
+	EntersPlayAction     *Action
```

Comment:
Can this be something other than an `Action`? Right now, actions are just things done by players of the game. We could do things like check in TakeAction that the action being taken is actually one of the valid actions. This seems like it should be something more like an Effect, or a triggered ability more specifically. This is also another dependency cycle that makes encoding tricky.